scripts/superdesk/upload/image-crop-directive.js

```diff
@@ -162,5 +269,94 @@
             }
         };
     }])
-    ;
+    .directive('sdImagePoint', ['$rootScope', function($rootScope) {
+        return {
+            scope: {
+                src: '=',
+                point: '=',
+                onChange: '&'
+            },
+            link: function(scope, elem) {
+                var img;
+                var pointElem;
+                var crossLeft;
+                var crossTop;
+                var crossBottom;
+                var crossRight;
+
+                elem.css({'position': 'relative'});
+
+                scope.$watch('src', function(src) {
+                    refreshImage(src);
+                });
+
+                scope.$watch('point', function() {
+                    if (img && pointElem) {
+                        drawPoint();
+                    }
+                }, true);
+
+                function refreshImage(src) {
+                    elem.empty();
+
+                    img = new Image();
+                    img.onload = function() {
+                        elem.append(img);
+                        drawPoint();
+                    };
+                    img.addEventListener('click', function(event) {
+                        elem.addClass('transition-on');
+                        scope.point.x = Math.round(event.offsetX * 100 / img.width) / 100;
+                        scope.point.y = Math.round(event.offsetY * 100 / img.height) / 100;
+                        scope.onChange();
+                        scope.$apply();
+                        $rootScope.$broadcast('poiUpdate', scope.point);
+                    });
+                    img.src = src;
+
+                    pointElem = angular.element('<div class="poi__cursor"></div>');
+                    elem.append(pointElem);
+                    pointElem.css({
+                        'position': 'absolute',
+                        'z-index': 10000
+                    });
+
+                    crossLeft = angular.element('<div class="poi__cross-left"></div>');
+                    elem.append(crossLeft);
+                    crossRight = angular.element('<div class="poi__cross-right"></div>');
+                    elem.append(crossRight);
+                    crossTop = angular.element('<div class="poi__cross-top"></div>');
+                    elem.append(crossTop);
+                    crossBottom = angular.element('<div class="poi__cross-bottom"></div>');
+                    elem.append(crossBottom);
+
+                }
+
+                function drawPoint() {
+                    pointElem.css({
+                        left: (scope.point.x * img.width) - 15,
+                        top: (scope.point.y * img.height) - 15
+                    });
```

Comment:
I can't do that without understanding the logic behind :)
@eozan does it worth it ?